src/shared/ethtool-util.c

```diff
@@ -50,6 +51,8 @@ DEFINE_STRING_TABLE_LOOKUP(port, NetDevPort);
 DEFINE_CONFIG_PARSE_ENUM(config_parse_port, port, NetDevPort, "Failed to parse Port setting");
 
 static const char* const netdev_feature_table[_NET_DEV_FEAT_MAX] = {
+        [NET_DEV_FEAT_RX]   = "rx-checksum",
+        [NET_DEV_FEAT_TX]   = "tx-checksum-",
```

Comment:
still would like to see a quick comment here that explains the situation, i.e. what the "-" at the end means. Could be as short as `/* The suffix "-" means any feature beginning with "tx-checksum-" */